meteor/client/ui/RundownView/RundownTiming/BreakWallTime.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames'
+import React from 'react'
+import { SegmentUi } from '../../SegmentTimeline/SegmentTimelineContainer'
+import { RundownUtils } from '../../../lib/rundown'
+import { useTranslation } from 'react-i18next'
+
+interface IBreakWallTimeProps {
+	segment: SegmentUi
+	className?: string
+	labelClassName?: string
+}
+
+/**
+ * A presentational component that will render a counter that will show the wall time that a break starts at.
+ * (Or ends at, if there's no start time but an end time is provided.)
+ * @function BreakWallTime
+ * @extends React.Component<IBreakWallTimeProps>
+ */
+export const BreakWallTime = function BreakWallTime(props: IBreakWallTimeProps): JSX.Element | null {
+	const { t } = useTranslation()
+	const value = props.segment.segmentTiming?.expectedStart ?? props.segment.segmentTiming?.expectedEnd
+	const passedExpectedStart =
+		props.segment.segmentTiming?.expectedStart && Date.now() > props.segment.segmentTiming?.expectedStart
```

Comment:
This should use `getCurrentTime()` as that's our only source of "current time" that is synced across Sofie clients.

Also, I don't think this will currently react to the time flowing, so I suspect you want to use `useCurrentTime()`.